bundles/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/protocol/API.java

```diff
@@ -319,12 +320,15 @@ public boolean sendCommand(APICode apiCode, String... apiData) {
                 break;
             case CommandOutputControl: /* 020 */
                 if (apiData[0] == null || !apiData[0].matches("[1-8]")) {
-                    logger.error("sendCommand(): Partition number must be a single character string from 1 to 8, it was: " + apiData[0]);
+                    logger.error(
+                            "sendCommand(): Partition number must be a single character string from 1 to 8, it was: "
+                                    + apiData[0]);
                     break;
                 }
 
                 if (apiData[1] == null || !apiData[1].matches("[1-4]")) {
-                    logger.error("sendCommand(): Output number must be a single character string from 1 to 4, it was: " + apiData[1]);
+                    logger.error("sendCommand(): Output number must be a single character string from 1 to 4, it was: "
+                            + apiData[1]);
```

Comment:
Not strictly necessary since the likelihood of errors being configured to be logged is very high, but this could be changed to use parameterized logging instead of string concatenation.